resources/scripts/components/dashboard/AccountOverviewContainer.tsx

```diff
@@ -1,4 +1,6 @@
 import * as React from 'react';
+import { useEffect } from 'react';
```

Comment:
@DaneEveritt Not sure if this is right, still trying to learn what I'm exactly doing in react.

Reply:
You can actually just do `import React, { useEffect } from 'react';` which will do the same thing. The `import * as React from 'react';` was from before I made some changes in typescript that remove the need for that `*` syntax.

You could also use `React.useEffect()` in this case, but I don't do that anywhere, so my first example in this response is the best answer.

Reply:
Changed app.tsx to use `import React, { useEffect } from 'react';`